kube/deploy.sh

```diff
@@ -1,12 +1,13 @@
 #!/bin/bash
 set -euo pipefail # make bash quit if something weird happens
 
-export KUBE_NAMESPACE=${ENVIRONMENT}
+export KUBE_NAMESPACE=${DRONE_STEP_NAME}
 export VERSION=${VERSION:-latest}
+export KUBE_SERVER=https://kube-api-notprod.notprod.acp.homeoffice.gov.uk
```

Comment:
If we have a safe one now, we can deploy to prod. Makes sense to set that up now.

Reply:
We'll do it in a subsequent commit once we've got confidence in the toolbox, I think. Let's leave it for now.